packages/zosfiles/src/constants/ZosFiles.messages.ts

```diff
@@ -439,18 +412,8 @@ export const ZosFilesMessages: { [key: string]: IMessageDefinition } = {
      * @type {IMessageDefinition}
      * @deprecated
```

Comment:
Please remove the `@deprecated` line since we decided this message was erroneously marked for deprecation.